common/transport/amqp/src/amqp.ts

```diff
@@ -3,8 +3,9 @@
 
 'use strict';
 
-import * as amqp10 from 'amqp10';
+import rhea = require('rhea');
```

Comment:
>require('rhea'); [](start = 14, length = 16)

nit: with the new type definitions that Amar contributed you should be able to have a standard typescript import statement